### PR TITLE
Vignette documentation

### PR DESCRIPTION
**Pull request by JedGrabman**
Add instructions to the getting started documentation to explain how to include vignettes in the package. This partially addresses issue #102

### CODE REVIEW

File: R-packages/covidcast/vignettes/covidcast.Rmd [hunk 1]
```diff
@@ -25,6 +25,17 @@ devtools::install_github("cmu-delphi/covidcast", ref = "main",
                          subdir = "R-packages/covidcast")
 ```
 
+Building vignettes takes a substantial amount of time, so they are not included
+by default. If you wish to include vignettes, use this modified command
```
Comment (capnrefsmmat) [inline R-packages/covidcast/vignettes/covidcast.Rmd:29]:
```suggestion
Building the vignettes, such as this Getting Started guide, takes a substantial amount of time. They are not included
in the package by default. If you wish to include vignettes, use this modified command:
```

with appropriate line-wrapping applied

**Review by capnrefsmmat — CHANGES_REQUESTED** [review]
Looks good, provided you tested the command; just one requested change to provide a bit of context so readers know why they'd want the vignettes

File: R-packages/covidcast/vignettes/covidcast.Rmd (hunk 1 above)
Comment (JedGrabman) [reply R-packages/covidcast/vignettes/covidcast.Rmd]: I tested the command locally and was able to build the vignettes. Phrasing has been updated.

**Comment by capnrefsmmat** [comment]
Great, thanks. We can either rebuild the documentation now (use `pkgdown::build_site()` and commit the HTML changes) or merge this without the updated HTML, so that the change will be included the next time we update the docs. Probably easiest to rebuild now, if you can.

**Comment by JedGrabman** [comment]
Went ahead and built now. It took in some other minor documentation changes that hadn't been built yet as well. Additionally, pkgdown updated from 1.5.1 to 1.6.1 leading to some minor HTML changes.

**Comment by capnrefsmmat** [comment]
Looks good, thanks.

**Pull request merged**